terraform/deployments/github/repos.yml

```diff
@@ -628,6 +628,9 @@ repos:
       additional_contexts:
         - Test Ruby
 
+  search-api-v2-dataform:
+    homepage_url: "https://docs.publishing.service.gov.uk/apps/search-api-v2-dataform.html"
```

Comment:
Should this url be to `apps` or `repos`? E.g. https://docs.publishing.service.gov.uk/repos/ga4-dataform.html